themes/lucene/templates/htaccess.template

```diff
@@ -126,19 +91,15 @@ RedirectMatch Permanent ^/nutch/(.*)$ https://nutch.apache.org/$1
 # Lucy has moved, site strcuture has changed, do don't use $1 in redirect...
 RedirectMatch Permanent ^/lucy/(.*)$ https://lucy.apache.org/
 
-# promote Solr to TLP
-# TODO: This links to the staging site for new Solr - change the redirect to TLP page
-#RedirectMatch Permanent ^/solr/(.*)$ https://solr.apache.org/$1
-RedirectMatch Permanent ^/solr/(.*)$ https://lucene-solrtlp.staged.apache.org/$1
-
+# Solr moved to TLP Feb 2021
```

Comment:
Simplified as two rules - needs separate rule for the ATOM feed

Reply:
This also simplifies the maintenance. Lucene webserver maintainer should not need to maintain those rules.

Reply:
Did not know about that feeds URL. +1